### PR TITLE
New Themes 

### PR DESCRIPTION
**Pull request by pchy-ynda**
### FIX
Added 2 new themes: Dark-gray and Midnight-blue

### Issue
Fixes the Issue #88 

### Notes to reviewer
I have added 2 new themes to the app, dark-gray and midnight-blue versions that are minimalistic with a good contrast for the users. Please let me know if you'd like to fix anything.

### CODE REVIEW

**Review by JhohannesK — CHANGES_REQUESTED** [review]

File: package-lock.json
[the remark this answers is not in the document]
Comment (pchy-ynda) [reply package-lock.json:1]: Sorry, I hope I fixed it now. Please let me know if its still not working.

**Comment by JhohannesK** [comment]
You have conflict on your branch. Kindly pull from `preview` branch and resolve the conflict.
Since someone else also contributed to the same files, you can accept both changes, thus, changes from you and from the other person

**Comment by pchy-ynda** [comment]
> You have conflict on your branch. Kindly pull from `preview` branch and resolve the conflict. Since someone else also contributed to the same files, you can accept both changes, thus, changes from you and from the other person

I tried to resolve the conflicts by manually editing the files and to accept both changes like you mentioned. May I know what went wrong?

**Comment by JhohannesK** [comment]
> > You have conflict on your branch. Kindly pull from `preview` branch and resolve the conflict. Since someone else also contributed to the same files, you can accept both changes, thus, changes from you and from the other person
> 
> I tried to resolve the conflicts by manually editing the files and to accept both changes like you mentioned. May I know what went wrong?

You must have done something wrong.
image

**Comment by JhohannesK** [comment]
So revert the commit you just added.
* pull down the changes from preview `preview`: `git pull origin preview --ff`
* I'm guessing you're using vscode. After pulling, if there are any conflict, vscode also shows it in the files and provides ways of resolving the conflict.

if you're still having issues, hmu on discord.
username: `jhohannes`

**Comment by pchy-ynda** [comment]
> So revert the commit you just added.
> 
> * pull down the changes from preview `preview`: `git pull origin preview --ff`
> * I'm guessing you're using vscode. After pulling, if there are any conflict, vscode also shows it in the files and provides ways of resolving the conflict.

Sorry this is my first time contributing, I will fix it as soon as I can,

**Comment by JhohannesK** [comment]
No worries,
Ask all the questions you need.
You can do this..
________________________________

**Pull request closed without merging**